support-services/src/test/scala/com/gu/support/pricing/PriceSummaryServiceSpec.scala

```diff
@@ -55,6 +53,11 @@ class PriceSummaryServiceSpec extends FlatSpec with Matchers {
     guardianWeekly(UK)(Domestic)(NoProductOptions)(Annual)(GBP).promotions
       .find(_.promoCode == GuardianWeekly.AnnualPromoCode).get.discountedPrice shouldBe Some(135.00)
 
+    // SixWeekly should have the 6 for 6 promotion and the discount
+    guardianWeekly(UK)(Domestic)(NoProductOptions)(SixWeekly)(GBP).promotions.size shouldBe 2
+    guardianWeekly(UK)(Domestic)(NoProductOptions)(SixWeekly)(GBP).promotions
+      .find(_.promoCode == GuardianWeekly.SixForSixPromoCode).get.introductoryPrice.get.price shouldBe 6
```

Comment:
There's a trait called `OptionValues` in ScalaTest which we should prefer here, which will give you a `.value` method to use on testing options. It will wrap exceptions  thrown by`.get` in the right way.